normcap/screengrab/permissions.py

```diff
@@ -222,9 +222,9 @@ def has_screenshot_permission() -> bool:
     logger.debug("Checking screenshot permission")
     if sys.platform == "darwin":
         return _macos_has_screenshot_permission()
-    if sys.platform == "linux" and not system_info.has_wayland_display_manager():
+    if (sys.platform == "linux" or sys.platform.startswith("freebsd")) and not system_info.has_wayland_display_manager():
```

Comment:
```suggestion
    if (sys.platform in ["linux", "freebsd"]) and not system_info.has_wayland_display_manager():
```
As `.startswith()` is [not necessary for Python 3.8+](https://docs.python.org/3/library/sys.html#sys.platform)) and NormCap requires at least Python 3.9, would you mind simplifying the syntax a bit? 

Same for the other conditions below.

Reply:
Hi @dynobo ,

The suggested change isn't equivalent, since the platform string is 'freebsd14'.

Reply:
Ah, you are right! Only for AIX the version is not reported anyore...